packages/at_root_server/lib/src/server/at_root_config.dart

```diff
@@ -1,89 +1,117 @@
+    }
+    return _httpsEnabled;
+  }
+
+  static bool get debugLog {
     var result = _getBoolEnvVar('debugLog');
     if (result != null) {
       return result;
     }
-    if (ConfigUtil.getYaml() != null &&
-        ConfigUtil.getYaml()!['log'] != null &&
-        ConfigUtil.getYaml()!['log']['debug'] != null) {
-      return ConfigUtil.getYaml()!['log']['debug'];
+    if (yaml != null && yaml!['log'] != null && yaml!['log']['debug'] != null) {
+      return yaml!['log']['debug'];
     }
     return _debugLog;
   }
 
-  static String? get privateKeyLocation {
-    if (_envVars.containsKey('privateKeyLocation')) {
-      return _envVars['privateKeyLocation'];
+  static String get privateKeyLocation {
+    if (envVars.containsKey('privateKeyLocation')) {
+      return envVars['privateKeyLocation']!;
     }
-    if (ConfigUtil.getYaml() != null &&
-        ConfigUtil.getYaml()!['security'] != null &&
-        ConfigUtil.getYaml()!['security']['privateKeyLocation'] != null) {
-      return ConfigUtil.getYaml()!['security']['privateKeyLocation'];
+    if (yaml != null &&
+        yaml!['security'] != null &&
+        yaml!['security']['privateKeyLocation'] != null) {
+      return yaml!['security']['privateKeyLocation'];
     }
     return _privateKeyLocation;
   }
 
-  static String? get certificateChainLocation {
-    if (_envVars.containsKey('certificateChainLocation')) {
-      return _envVars['certificateChainLocation'];
+  static String get certificateChainLocation {
+    if (envVars.containsKey('certificateChainLocation')) {
+      return envVars['certificateChainLocation']!;
     }
-    if (ConfigUtil.getYaml() != null &&
-        ConfigUtil.getYaml()!['security'] != null &&
-        ConfigUtil.getYaml()!['security']['certificateChainLocation'] != null) {
-      return ConfigUtil.getYaml()!['security']['certificateChainLocation'];
+    if (yaml != null &&
+        yaml!['security'] != null &&
+        yaml!['security']['certificateChainLocation'] != null) {
+      return yaml!['security']['certificateChainLocation'];
     }
     return _certificateChainLocation;
   }
 
-  static bool? get useSSL {
+  static bool get useSSL {
     var result = _getBoolEnvVar('useSSL');
     if (result != null) {
       return result;
     }
-    if (ConfigUtil.getYaml() != null &&
-        ConfigUtil.getYaml()!['security'] != null &&
-        ConfigUtil.getYaml()!['security']['useSSL'] != null) {
-      return _useSSL = ConfigUtil.getYaml()!['security']['useSSL'];
+    if (yaml != null &&
+        yaml!['security'] != null &&
+        yaml!['security']['useSSL'] != null) {
+      return yaml!['security']['useSSL'];
     }
     return _useSSL;
   }
 
   static bool? _getBoolEnvVar(String envVar) {
-    if (_envVars.containsKey(envVar)) {
-      (_envVars[envVar]!.toLowerCase() == 'true') ? true : false;
+    if (envVars.containsKey(envVar)) {
+      return (envVars[envVar]!.toLowerCase() == 'true') ? true : false;
```

Comment:
fix: Previously this piece of code wasn't returning the value it found, so it dropped through and returned null